src/styles/main.scss

```diff
@@ -1,3 +1,49 @@
+html {
+  font-family: Arial, sans-serif;
+  font-size: 100px;
+}
+
 body {
   margin: 0;
 }
+
+.container {
+  display: grid;
+  grid-template-columns: repeat(3, minmax(300px, 1fr));
+  justify-content: center;
+  grid-auto-flow: row;
+}
+
+@media (max-width:899px) {
+  .container {
+    grid-template-columns: repeat(2, minmax(300px, 1fr));
+  }
+}
+
+@media (max-width:599px) {
+  .container {
+    grid-template-columns: minmax(300px, 1fr);
+  }
+}
+
+.container__block {
+  height: 300px;
+  display: grid;
+  justify-content: center;
+  align-content: center;
+  color: white;
+}
+
+$colors:
+  "1" #f00,
```

Comment:
That's a good option. You can also think about using RGB in a loop

Reply:
> That's a good option. You can also think about using RGB in a loop

I used ColorZilla to find the correct color
Is it necessary to use RGB?